SimMuon/Configuration/python/customiseForMuonAging.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Hi @davidlange6 , sure, I will proceed addressing the comments from @ghellwig , making the customization upgrade-specific and adding comments in the file explaining the non reproducibility. Do you want that, besides comments, the function in `customiseForMuonAging.py` also print come warning abut reproducibility when called?